src/osd/OSD.cc

```diff
@@ -4124,7 +4124,7 @@ void OSD::heartbeat_entry()
   while (!heartbeat_stop) {
     heartbeat();
 
-    double wait = .5 + ((float)(rand() % 10)/10.0) * (float)cct->_conf->osd_heartbeat_interval;
+    double wait = .5 + (double)(rand() / (double)RAND_MAX) * (double)cct->_conf->osd_heartbeat_interval;
```

Comment:
Have you actually observed this being a problem?  I'm not convinced that this is a meaningful change.